TreeDB/collections/vector_index.go

```
@@ -1867,6 +1932,149 @@ func (idx *VectorIndex) insertVectorLocked(documentID []byte, vector []float32)
+					return err
+				}
+			}
+			start = end
+			continue
+		}
+		entry, maxLevel := idx.entry, idx.maxLevel
+		plans := make([]vectorIndexFrozenPrefixInsert, end-start)
+		errs := make([]error, len(plans))
+		var wg sync.WaitGroup
+		for worker := 1; worker < len(plans); worker++ {
+			worker := worker
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "worker" can be deleted (Go 1.22+)

(copyloopvar)